icons-gallery/src/main.ts

```diff
@@ -0,0 +1,32 @@
+import { Devvit } from "@devvit/public-api";
+
+import { IconsGallery } from "./components/IconsGallery.js";
+import { Preview } from "./components/Preview.js";
+
+Devvit.debug.emitSnapshots = true;
```

Comment:
Dang, I didn't see that we had actually built this. I suggest removing since users will likely be confused.